docs/docs-source/docs/modules/get-started-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,73 @@
+= Run a Sample Cloudflow Application on GCP Marketplace
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Setting up CLI
+
+With Cloudflow installed on a GKE cluster through the Marketplace, you can interact with Cloudflow via its `kubectl` plugin: `kubectl cloudflow`.
+If you haven't already, download the CLI from the https://github.com/lightbend/cloudflow/releases/tag/v2.0.0[Github release page].
+Unzip the downloaded archive and then put the output file `kubectl-cloudflow` under `/usr/local/bin/`.
```

Comment:
We already have instructions for setting up the CLI here: https://cloudflow.io/docs/current/get-started/prepare-development-environment.html#_download_and_install_the_cloudflow_cli